src/components/Navigation/Navigation.jsx

```diff
@@ -37,32 +32,26 @@ export default function Navigation(){
 		}
 	];
 
-
-	const links       = linkData.map(RENDER.item);
-	const ariaOpen    = open.toString();
+	const links      	  = linkData.map(RENDER.item);
+	const ariaExpanded    = expanded.toString();
 
 	return (
-		<nav>
-			<button 
-				role="switch"
-				aria-checked={ ariaOpen }
-				aria-controls="navigation__links"
-				onClick={ toggleOpen }
-			>
-				<span>
-					Menu
-				</span>
-				<span>
-					Close
-				</span>
-			</button>
+		<nav className={ s.wrapper }>
+			<NavProvider>
+				<NavToggle 
+					setExpanded={ setExpanded }
```

Comment:
```suggestion
```
As a rule of thumb, now that you know about contexts, there's not a need to be passing props down like this anymore 👌